charts/model-engine/templates/service_template_config_map.yaml

```diff
@@ -472,6 +472,12 @@ data:
           unsafeSsl: "false"
           databaseIndex: "${REDIS_DB_INDEX}"
       {{- end }}
+      - type: prometheus
```

Comment:
curious how redis and istio metrics work together here?

Reply:
it'll take whichever metric gives the higher value, in practice the redis metric only gives a fast 0-1 signal (since the length of the redis queue is at most 1 barring weird race conditions) and the prometheus metric (which is identical to the 1-N hpa scaling metric) handles the 1-N condition